target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1,6 @@
+/usr/local/google/home/dikshamodi/hyperlocal-gpay-spot/src/main/java/com/example/hyperlocal/MerchantController.java
```

Comment:
Are these autogenerated ? If yes, put them in .gitignore.

Same for .class files

Reply:
I've removed the autogenerated and binary files